docs/release-notes/Percona-Everest-1.2.0-(2024-09-09).md

```diff
@@ -0,0 +1,166 @@
+# What's new in Percona Everest 1.2.0
+
+To begin your journey with Percona Everest, check out the [Quickstart Guide for Percona Everest](../quickstart-guide/quick-install.md).
+
```

Comment:
We need a big warning sign at the beginning letting users know about the backward incompatible changes.

Reply:
Yeah.

Reply:
Done.